cg/meta/report/report_api.py

```diff
@@ -115,7 +115,7 @@ def get_cases_without_delivery_report(self, pipeline: Pipeline) -> list[Case]:
             :MAX_ITEMS_TO_RETRIEVE
         ]
         for analysis_obj in analyses:
```

Comment:
```suggestion
        for analysis in analyses:
```
Would be cool to catch this, ignore if out of scope.

Reply:
Yeah, I'm not brave enough to rename these here - can do it separately :)